src/sage/arith/misc.py

```diff
@@ -1970,12 +1970,22 @@ def xgcd(a, b):
 
```

Comment:
What about the `NOTE` above? Is that still accurate?

Reply:
I'm not 100% sure about this, but I believe so. It would be great if someone can double check.

Reply:
I mean, do you want to remove the reference to `a` and `b` and just say "the elements"? Because now b might be None and a a list.

Reply:
Oh yes you're complete right. I'll update.